docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -14,8 +14,27 @@
     
     - set_fact:
         deploy_config: "{{ deployment_config }}"
+        wait_for_dm_unlock: " {{ wait_for_dm_unlock| default(1) }}"
       when: deployment_config is defined
 
+    - block:
+      # Information to be used at final DM checks.
+        - name: Get host name
+          shell: |
+            source /etc/platform/openrc; hostname
+          register: get_host_name 
+
+        - name: Get region name
```

Comment:
The subcloud's region name is not necessarily "subcloud*",  if you want to check if the host is a subcloud's node, better to check with"      shell: source /etc/platform/platform.conf; echo $distributed_cloud_role"

Reply:
Makes sense. Testing the change. Thanks Yuxing!

Reply:
Done.